### PR TITLE
Tweak wording

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
I spoke to @bkeepers and @kytrinyx about the wording here and they had some great suggestions that I've made here.

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016 OSS Friday
+Copyright (c) 2016 Open Source Friday
```
Comment (zverok) [inline LICENSE.txt]: Year ;)

Comment (maxcell) [inline LICENSE.txt]: Nice catch!

Comment (maxcell) [inline LICENSE.txt]: I think that they started the project in 2016 though so arguable on when they want to do the year?

Comment (MikeMcQuaid) [inline LICENSE.txt]: Thanks! Removing the year entirely because they aren't legally required and require infinite updating 😆

**Comment by Thetoxicarcade** [comment]
Cool! this had reviewing!

File: LICENSE.txt (hunk 1 above)
Comment (bkeepers) [inline LICENSE.txt]: FWIW, "Open Source Friday" isn't an entity that can own a copyright. "Open Source Friday [contributors](https://github.com/ossfriday/ossfriday/graphs/contributors)" is probably more appropriate.

Comment (bkeepers) [inline LICENSE.txt]: nevermind, I see that's already been updated.

**Pull request merged**